test/fixtures/x-summary-type.yaml

```diff
@@ -0,0 +1,17 @@
+swagger: '2.0'
+info:
+  title: Resource summary as incorrect type coerces to string
+  version: '1.0'
+paths:
+  '/boolean':
+    x-summary: true
```

Comment:
`true` is a boolean literal and previously would create a BooleanElement.

Reply:
Agree